config/ktlint/baseline.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file is automatically generated by the `ktlintGenerateBaseline` task and is used by `ktlint` to essentially ignore known style violations, while still allowing you to use the tool to detect new ones.
For example, we have some package names that contain an underscore ("com.pubnub....objects_api") which violates the ktlint rules and fails the build. We can't change the package name (it's a breaking change), so instead we can generate a baseline that contains those known violations and commit it to the project. We will still be notified (and fail the build) when new underscore_package_names are added, but it will allow the existing one to be there.

Reply:
Thank you for clarification.